src/index.html

```diff
@@ -1,13 +1,467 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="page" lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/logo.png">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+
+  <body class="page__body body">
```

Comment:
```suggestion
  page__body body
```
элмент не может быть вместе с родительским блоком